### ISSUE TITLE
down doesn't respect profiles

### ISSUE THREAD

**Issue by BretFisher** [docker/compose-cli#1698]
**Description**

If using `--profile` in commands, `down` won't try to stop the containers in that profile, and will get a "failed to remove network" error because containers are still running that use those networks.

**Steps to reproduce the issue:**
1. `docker compose --profile profile-name up`
2. `docker compose --profile profile-name down`
3. See error "failed to remove network xxxxx"
4. `docker compose --profile profile-name ps` verifies all profile-specific services are still up.

**Describe the results you received:**
`down` ignores any services created with that profile.

**Describe the results you expected:**
`down` conforms to the same usage of other commands with profiles.

**Output of `docker version`:**

```
$docker compose version
Docker Compose version 2.0.0-beta.1
```

```
$docker version
Client:
 Cloud integration: 1.0.14
 Version:           20.10.6
 API version:       1.41
 Go version:        go1.16.3
 Git commit:        370c289
 Built:             Fri Apr  9 22:46:57 2021
 OS/Arch:           darwin/amd64
 Context:           default
 Experimental:      true
```

**Output of `docker context show`:**  

```
default
```

**Comment by ndeloof** [comment]
This is probably a side-effect for an issue fixed by https://github.com/docker/compose-cli/pull/1641
2.0.0-beta.2 (1.0.15) has been released, could you please give it a try?

**Comment by BretFisher** [comment]
Looks fixed with `2.0.0-beta.3` thanks! Closing.
